client/composables/axios/useAxiosMiddlewares.ts

```diff
@@ -0,0 +1,37 @@
+import type { AxiosMiddlewares } from './types'
+
+export const skipRefreshRoutes = ['/refresh', '/login', '/users/signup', '/upload/token']
+
+export function useAxiosMiddlewares(): ComputedRef<AxiosMiddlewares> {
+  return computed(() => ({
+    requestMiddlewares: [
+      {
+        route: skipRefreshRoutes,
+        negateRoute: true,
+        onFulfilled: (config) => {
+          const tokens = getTokensFromLocalStorage()
+          if (tokens) {
+            config.headers.Authorization = `Bearer ${tokens.accessToken}`
+            config.headers['Access-Control-Allow-Origin'] = '*'
```

Comment:
Hmm is this CORS sh*t really necessary?

Reply:
it is for some browsers :pain: funny thing is that it actually is a restriction from the browser, not the server. 🙃

Reply:
haha (cringe)